fastify.js

```diff
@@ -210,9 +210,9 @@ function build (options) {
         new State(req, res, params, context),
         middlewareCallback
       )
-    } else {
-      middlewareCallback(null, new State(req, res, params, context))
+      return
```

Comment:
Can you leave this as it was before?

Reply:
For me is the same.

Is there a reason to choose one way instead of the other one?

Reply:
I agree with @delvedor for these.

Reply:
@mcollina probably can give you a better explanation.
If possible we should try to avoid the *return early* cases, because they create a more JMP than needed and V8 is not very happy about that.
Often try to optimize that is not worth the amount of work, but where is easy to do... why not?

Reply:
done